shell/platform/android/test/io/flutter/plugin/platform/PlatformViewWrapperTest.java

```diff
@@ -234,14 +233,15 @@ public void sendAccessibilityEvents() {
     assertTrue(eventSent);
   }
 
-  @Implements(View.class)
-  public static class ShadowView {}
-
   @Implements(ViewGroup.class)
-  public static class ShadowViewGroup extends org.robolectric.shadows.ShadowView {
+  public static class ShadowViewGroup extends org.robolectric.shadows.ShadowViewGroup {
     @Implementation
-    public boolean requestSendAccessibilityEvent(View child, AccessibilityEvent event) {
+    protected boolean requestSendAccessibilityEvent(View child, AccessibilityEvent event) {
       return true;
     }
   }
+
+  @Implements(FrameLayout.class)
+  public static class ShadowFrameLayout
+      extends io.flutter.plugin.platform.PlatformViewWrapperTest.ShadowViewGroup {}
```

Comment:
Essentially, the issue here was that when you create a custom Robolectric shadow, it needs to imitate the object hierarchy of the object you are trying to shadow. Here (and in `FlutterMutatorViewTest`), we are actually need to shadow `FrameLayout`, although it makes a call to its parent for the method whose behavior we want to mock (`ViewGroup`).